sql/catalyst/src/main/scala-2.12/org/apache/spark/sql/catalyst/expressions/ExpressionSet.scala

```diff
@@ -27,6 +27,10 @@ object ExpressionSet {
     expressions.foreach(set.add)
     set
   }
+
```

Comment:
We should apply the same change in `ExpressionSet` under the `scala-2.13` source tree. @dbaliafroozeh can you open a followup  PR?

Reply:
@cloud-fan good catch, I thought I already delete the ExpressionSet in 2.13. Note that we don't want it anymore as ExpressionSet doesn't extend Set anymore. I'll open a followup PR for that.